### PR TITLE
Introduce html file as root page

### LINKED ISSUE #209: Make root page nicer
As we are now moving to production, it would be nice to come up with a useful and nice contents of the root page. Now it just contains a test message to confirm that server is up.

Maybe tell a little about oneseismic, mention link to the swagger docs, explain how to find storage accounts and sas?
Or just redirect to swagger docs?

### PR DESCRIPTION
**Pull request by yngve793**
Message for the root in the web server is updated with user relevant information. 
Message is moved out into separate html file. 
docs/metadata.md file is added.

Comments in endpoint.go are added as otherwise the function must be split into multiple function calls to introduce function names with the same information. Also it is unclear if this is the best Go code to preform the desired task. Suggestions are welcome. 

References to "cloudflare.com" are used to present the Python code. Do we need a license for this?

For completeness the docs/metadata.md file is added. It is satisfying to have one readme file for each endpoint. 

Close #209

### CODE REVIEW

File: docs/metadata.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
```
Comment (achaikou) [inline docs/metadata.md:1]: (Get) volumetric...

Other mds start with a verb.

File: docs/metadata.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename. 
+
+## Response
+On success (200) the response containing the VDS metadata.
```
Comment (achaikou) [inline docs/metadata.md:6]: *Content-Type: application/json*
On success (200) the json response contains metdata. See the Metadata model.

?

File: docs/metadata.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename.
```
Comment (achaikou) [inline docs/metadata.md:3]: Maybe not VDS file, but VDS data?

Some of the information we return doesn't belong to vds file - we calculate it.

File: docs/metadata.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename. 
+
+## Response
+On success (200) the response containing the VDS metadata.
+
+### Metadata part
```
Comment (achaikou) [inline docs/metadata.md:8]: There is no part here, it is not a multipart response.
I think kill this at all. It just repeats itself.

**Review by yngve793 — COMMENTED** [review]
Lots of changes here. Most suggestions are taken into account. 
The text in the index.md has been rephrased, so there might very well be new stuff there.

File: docs/metadata.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Returns metadata describing the VDS data
+
+Retrieve basic information related to the VDS data such as axis definitions, overall shape and size, etc."
```
Comment (achaikou) [inline docs/metadata.md:3]: Bad " at the end.

File: docs/metadata.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Returns metadata describing the VDS data
+
+Retrieve basic information related to the VDS data such as axis definitions, overall shape and size, etc." 
+
+## Response
+Content-Type: application/json
```
Comment (achaikou) [inline docs/metadata.md:6]: Italic from my comment didn't copy well apparently :smile: 
This is italic on other endpoints.

File: api/index.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## API documentation
+Swagger documentation is available <a href="swagger/index.html">here</a>. 
+
+## Source code
+VDS-slice is an open project and the code is available <a href="https://github.com/equinor/vds-slice">here</a>.
+
+
+## A simple python example
+```
```
[the remark this answers is not in the document]
Comment (yngve793) [reply api/index.md:22]: Thanks, very cool. :-)

File: api/index.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+<a href="https://community.opengroup.org/osdu/platform/domain-data-mgmt-services/seismic/open-vds">Open VDS</a> is an open source library designed for cloud storage, where the data set is partitioned into sub-cubes of fixed size in each dimension. One consequence of this is a normalization of the access time in different dimensions. Open VDS provides the user with an interface for the global cube, and thus the underlying data structure is hidden from the user.
+
+In cases where the returned data set of a request do not span all dimensions there will be a significant overhead in the data retrieved from the cloud storage. Examples here are <i>fence</i> and <i>slice</i> requests. If the VDS is stored using sub-cubes of size 64, then fence and slice requests to Open VDS will retrieve roughly 64 times more data than the user requested. Reducing the sub-cube size will reduce the overhead at the expense of the time Open VDS used to manage the underlying data structure. Thus, there is a sweet sport balancing the two.
```
[the remark this answers is not in the document]
Comment (yngve793) [reply api/index.md]: 1. Rephrased
2. Yes, that is better. 
3. Thanks
4. Sure

I like to understand the business case of the project I am working on. I agree that most uses do care beyond response time, but is good to have a clear argument when justifying the spending.

File: api/index.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+In cases where the returned data set of a request do not span all dimensions there will be a significant overhead in the data retrieved from the cloud storage. Examples here are <i>fence</i> and <i>slice</i> requests. If the VDS is stored using sub-cubes of size 64, then fence and slice requests to Open VDS will retrieve roughly 64 times more data than the user requested. Reducing the sub-cube size will reduce the overhead at the expense of the time Open VDS used to manage the underlying data structure. Thus, there is a sweet sport balancing the two. 
+
+<a href="https://github.com/equinor/vds-slice">VDS-slice</a> addresses this issue by enabling the user to run the Open VDS instance in Azure or even better in the same data center as the cloud storage. In this way VDS-slice utilize the high internal data rate and only requested data is sent to the user. Thus, minimizing external data traffic.
```
[the remark this answers is not in the document]
Comment (yngve793) [reply api/index.md]: 1. Ok
2. Ok, rephrased.

**Review by yngve793 — COMMENTED** [review]
Changes applied

File: api/index.html [hunk 10]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html><html><head>
+      <title>index</title>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
```
[the remark this answers is not in the document]
Comment (yngve793) [reply api/index.html:5]: Using blackfriday or gomarkdown produces a html file without any style. It is not clear how these tools can be used to generate html and css at the same time. It is not expected that the index.md will be modified frequently. A description for generating the file is added to the top of the index.md.

File: api/index.md
Comment (achaikou) [inline api/index.md:1]: Price for having this reviewed from the start again :smiling_imp: 

Why are index.md and index.html in the api directory and not in docs?

File: api/index.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+In cases where the requested data set do not span all dimensions there will be a significant overhead in the data retrieved from the cloud storage. Examples here are <i>fence</i> and <i>slice</i> requests. If the VDS is stored using sub-cubes of size 64, then fence and slice requests to Open VDS will retrieve roughly 64 times more data than the user requested. Reducing the sub-cube size will reduce the overhead at the expense of the time Open VDS uses to manage the underlying data structure. Thus, there is a sweet spot balancing the two. 
+
+<a href="https://github.com/equinor/vds-slice">VDS-slice</a> addresses this issue by enabling the user to run the Open VDS instance in Azure or even better in the same data center as the cloud storage. In this way VDS-slice utilize(s) the data centers high internal data rate and minimize the external data traffic.
```
[the remark this answers is not in the document]
Comment (achaikou) [reply api/index.md]: Not fixed.

**Review by achaikou — COMMENTED** [review]
Btw, forgot to mention last time.
Please, for simple changes push fixups, do not do force-push.

With fixups I can check only stuff that you changed. Without fixups I start reviewing everything you done anew from the start.

File: api/index.md
Comment (yngve793) [reply api/index.md:1]: Moved.

**Comment by achaikou** [comment]
Great! Rebase, apply fixups and we are merging!

**Review by achaikou — APPROVED** [review]

**Pull request merged**